### DISCUSSION TITLE
Question about hosting a SaaS product blog on Github pages

### DISCUSSION THREAD

**Discussion by talonx** [community/community#126006] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

The GitHub pages TOS state that 
```
GitHub Pages is not intended for or allowed to be used as a free web-hosting service to run your online 
business, e-commerce site, or any other website that is primarily directed at either facilitating 
commercial transactions or providing commercial software as a service (SaaS).
```
Does hosting a blog for a SaaS product violate these terms of service? The product itself is obviously hosted elsewhere. There are no commercial transactions or software that will be provided from the blog content, but posts will talk about the product, among other things. I find the TOS a bit unclear on this, hence asking the experts here for advice.

[https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#prohibited-uses](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#prohibited-uses)

**Comment by LiteBrite82** [comment]
Hi @talonx 

I'm looking into it. Ping me back if you dont here from me by Monday. Great question!

**Reply by LiteBrite82** [reply]
do you have an example deployed anywhere?

**Reply by LiteBrite82** [reply, marked as answer]
OK, I have a more definitive answer:

"If the blog has content benefiting everyone including the ones not paying for their service, that could be ok. If it's pure content targeted at their customers; no." 

This is more of a "principal" situation. The hosting services is intended for open source development and learning; use that as your guide and you should be ok.

**Reply by talonx** [reply]
> do you have an example deployed anywhere?

https://blog.incidenthub.cloud/

**Reply by talonx** [reply]
> OK, I have a more definitive answer:
> 
> "If the blog has content benefiting everyone including the ones not paying for their service, that could be ok. If it's pure content targeted at their customers; no."
> 
> This is more of a "principal" situation. The hosting services is intended for open source development and learning; use that as your guide and you should be ok.

I'll use that as a guideline. Thank you.
